.docker/dever_apt_debs.txt

```diff
@@ -0,0 +1,4 @@
+# ROS tools
```

Comment:
For all 3 `*_apt_debs.txt`: Please add a comment about what should go into each of these files and when used